py/etrade-plan-pdf-tx-extract

```diff
@@ -3,20 +3,18 @@
 """
 A convenience script to extract transactions from PDFs downloaded from us.etrade.com
 """
-import ensure_in_venv
-
```

Comment:
Why? Getting rid of this will mean you have to run the script from inside of the venv. You can't just execute the script from anywhere anymore.

Reply:
Oh, sorry, I saw "unused import" and thought it was still unused. I'll put it back :)